<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateServerMainWindow</class>
 <widget class="QMainWindow" name="UpdateServerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ERP Update Server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../universal/SmartPay/src/payroll_resources.qrc">
    <normaloff>:/payroll/images/1403_Globe.png</normaloff>:/payroll/images/1403_Globe.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Connected Clients:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="lstClients">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Host Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>First Contact</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last Contact</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Detected Plugins</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="lstPlugins">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Plugin Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Description</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Release Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Version</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Log:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="lstLog"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMinimize"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../universal/SmartPay/src/payroll_resources.qrc"/>
 </resources>
 <connections/>
</ui>
